ant-node/src/put_validation.rs

```diff
@@ -646,22 +645,22 @@ impl Node {
             // FIXME: We would reach the MAX if the storecost is scaled up.
             let current_value = metrics_recorder.current_reward_wallet_balance.get();
             let new_value =
-                current_value.saturating_add(storecost.as_atto().try_into().unwrap_or(i64::MAX));
+                current_value.saturating_add(reward_amount.try_into().unwrap_or(i64::MAX));
             let _ = metrics_recorder
                 .current_reward_wallet_balance
                 .set(new_value);
         }
         self.events_channel()
-            .broadcast(crate::NodeEvent::RewardReceived(storecost, address.clone()));
+            .broadcast(crate::NodeEvent::RewardReceived(AttoTokens::from(reward_amount), address.clone()));
 
         // vdash metric (if modified please notify at https://github.com/happybeing/vdash/issues):
-        info!("Total payment of {storecost:?} atto tokens accepted for record {pretty_key}");
+        info!("Total payment of {reward_amount:?} atto tokens accepted for record {pretty_key}");
```

Comment:
this and couple of following logs/prints could be misleading if switched to say pay median or gradual emission scheme.
So, need to make sure the returned reward_amount is really the amount paid to this node directly, or the cost calculated according to the quoting metrics and pricing curve, or something else.

if self.evm_network().verify_data_payment() cann't return with something like (transaction_total_amount, paid_to_node, calculated_from_metrics), then we shall be careful on the wording here.